ivadomed/scripts/download_data.py

```diff
@@ -118,30 +127,58 @@ def unzip(compressed, dest_folder):
         raise
 
 
+def create_string(dict):
+    doc_str = ''
+    for i in range(len(dict.keys())):
+        if i == 0:
+            doc_str = doc_str + '`' + str(list(dict.keys())[i]) + ' <' + str(dict[list(dict.keys())[i]]["url"][0][:-4])\
+                      + '>`_ :' + str(dict[list(dict.keys())[i]]["description"]) + ' |br| '
+        else:
+            doc_str = doc_str + '`' + str(list(dict.keys())[i]) + ' <' + \
+                      str(dict[list(dict.keys())[i]]["url"][0][:-4]) + '>`_ : ' + \
+                      str(dict[list(dict.keys())[i]]["description"]) + ' |br| '
+    return doc_str
+
+
 def install_data(url, dest_folder, keep=False):
     """
-    Download a data bundle from a URL and install in the destination folder.
-    :param url: URL or sequence thereof (if mirrors).
-    :param dest_folder: destination directory for the data (to be created).
-    :param keep: whether to keep existing data in the destination folder.
-    :return: None
+    Download a data bundle from an URL and install it in the destination folder.
+
+    Usage example ::
+
+        ivadomed_download_data -d data_testing -o ivado_testing_data
+
+
+    Existing data bundle: |br|
+    %s
```

Comment:
the doc looks very cool! but i'm curious how it works?? (where is this "%s" defined???)

Reply:
here https://github.com/ivadomed/ivadomed/blob/482a6769a364b64f2051a6d0995b680133f45d78/ivadomed/scripts/download_data.py#L250

Reply:
What about using `install_data.__doc__.format(DICT_URL)`? Then you can rewrite `__doc__` to use named variables instead of bare `%s`s.

Reply:
I tried something similar and I might have made a mistake because it didn'T work. Would it be better practice?

Reply:
I forgot that you need to unpack the dict:

```
install_data.__doc__=install_data.__doc__.format(**DICT_URL)
```

But I didn't look at the rest of the context when I gave that suggestion.